Crash on startup on macOS Ventura on Apple Silicon

Running the `sync` command on an Apple Silicon Mac, running macOS Ventura, results in this crash:

```
Exception in thread "main" java.sql.SQLException: Error opening connection
	at org.sqlite.core.CoreConnection.open(CoreConnection.java:215)
	at org.sqlite.core.CoreConnection.<init>(CoreConnection.java:76)
	at org.sqlite.jdbc3.JDBC3Connection.<init>(JDBC3Connection.java:25)
	at org.sqlite.jdbc4.JDBC4Connection.<init>(JDBC4Connection.java:24)
	at org.sqlite.SQLiteConnection.<init>(SQLiteConnection.java:45)
	at org.sqlite.JDBC.createConnection(JDBC.java:114)
	at org.sqlite.JDBC.connect(JDBC.java:88)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:191)
	at com.squareup.sqldelight.sqlite.driver.JdbcSqliteDriver.<init>(JdbcSqliteDriver.kt:25)
	at com.squareup.sqldelight.sqlite.driver.JdbcSqliteDriver.<init>(JdbcSqliteDriver.kt:14)
	at com.jakewharton.dodo.DodoCommand.run(main.kt:118)
	at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:171)
	at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:180)
	at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:16)
	at com.github.ajalt.clikt.core.CliktCommand.parse(CliktCommand.kt:395)
	at com.github.ajalt.clikt.core.CliktCommand.parse$default(CliktCommand.kt:392)
	at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:410)
	at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:435)
	at com.jakewharton.dodo.Main.main(main.kt:28)
Caused by: java.lang.Exception: No native library is found for os.name=Mac and os.arch=aarch64. path=/org/sqlite/native/Mac/aarch64
	at org.sqlite.SQLiteJDBCLoader.loadSQLiteNativeLibrary(SQLiteJDBCLoader.java:333)
	at org.sqlite.SQLiteJDBCLoader.initialize(SQLiteJDBCLoader.java:64)
	at org.sqlite.core.NativeDB.load(NativeDB.java:56)
	at org.sqlite.core.CoreConnection.open(CoreConnection.java:211)
	... 19 more
```

--- a/build.gradle
+++ b/build.gradle
@@ -26,7 +26,7 @@ dependencies {
 	implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.4.0'
 	implementation 'com.github.ajalt.clikt:clikt:3.0.1'
 	implementation 'com.squareup.sqldelight:sqlite-driver:1.4.3'
-	implementation 'org.xerial:sqlite-jdbc:3.34.0'
+	implementation 'org.xerial:sqlite-jdbc:3.40.0.0'
 	implementation 'org.twitter4j:twitter4j-core:4.0.7'
 	implementation 'ch.qos.logback:logback-classic:1.2.3'
 

